tasks/sep/base-002-fp-upgrade/VALIDATION.md

```diff
@@ -0,0 +1,130 @@
+  **Meaning:** Clears the old and unused [L2OutputOracle slot](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/src/L1/SystemConfig.sol#L63). You can verify the correctness of the storage slots with `chisel`. Just start it up and enter the slot definitions as found in the contract source code.
+  ```
+  ➜ bytes32(uint256(keccak256("systemconfig.l2outputoracle")) - 1)
+  Type: bytes32
+  └ Data: 0xe52a667f71ec761b9b381c7b76ca9b852adf7e8905da0e0ad49986a0a6871815
+  ```
+
+### `0x49f53e41452C74589E85cA1677426Ba426459e85` (`OptimismPortalProxy`)
+
+Links:
+- [Sepolia Etherscan](https://sepolia.etherscan.io/address/0x49f53e41452c74589e85ca1677426ba426459e85)
+- [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/a06cb07264c985c041c0c01af57463ea88da264c/superchain/extra/addresses/sepolia/base.json#L8)
+
+State Changes:
+- **Key:**    `0x0000000000000000000000000000000000000000000000000000000000000038` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:**  `0x000000000000000000000000d6e6dbf4f7ea0ac412fd8b65ed297e64bb7a06e1` <br/>
+  **Meaning:**  Sets the `DisputeGameFactoryProxy` address in the proxy storage (0x38 is equivalent to 56). Note the `DisputeGameFactoryProxy` address should be the same address as the one set in the `SystemConfigProxy` state changes.
+
+- **Key:**    `0x000000000000000000000000000000000000000000000000000000000000003b` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:**  `0x0000000000000000000000000000000000000000000000006692e10f00000000` <br/>
+  **Meaning:** Sets the `respectedGameType` and `respectedGameTypeUpdatedAt` slot (0x3b is equivalent to 59).
+  The `respectedGameType` is 32-bits wide at offset 0 which should be set to 0 (i.e. [`CANNON`](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.4.0-rc.4/packages/contracts-bedrock/src/dispute/lib/Types.sol#L28)).
+  The `respectedGameTypeUpdatedAt` is 64-bits wide and offset by `4` on that slot. It should be equivalent to the unix timestamp of the block the upgrade was executed.
+  You can extract the offset values from the slot using `chisel`:
+  ```
+  ➜ uint256 x = 0x0000000000000000000000000000000000000000000000006692e10f00000000
+  ➜ uint64 respectedGameTypeUpdatedAt = uint64(x >> 32)
+  ➜ respectedGameTypeUpdatedAt
+  Type: uint64
+  ├ Hex: 0x
+  ├ Hex (full word): 0x6692e10f
+  └ Decimal: 1720901903
+  ➜ 
+  ➜ uint32 respectedGameType = uint32(x & 0xFFFFFFFF)
+  ➜ respectedGameType
+  Type: uint32
+  ├ Hex: 0x
+  ├ Hex (full word): 0x0
+  └ Decimal: 0
+  ```
+
+- **Key:**    `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc` <br/>
+  **Before:** `0x0000000000000000000000002d778797049fe9259d947d1ed8e5442226dfb589` <br/>
+  **After:**  `0x00000000000000000000000035028bae87d71cbc192d545d38f960ba30b4b233` <br/>
```

Comment:
Implementation is a verified [OptimismPortal2](https://sepolia.etherscan.io/address/0x35028bae87d71cbc192d545d38f960ba30b4b233#code) contract